### PR TITLE
refactor/130-clean-eslint-prettier

### PR DESCRIPTION
**Pull request by rochejul**
## Description

The ideas:
1) ensure to have all expected eslint dependencies
2) ensure to have only one eslint configuration
3) align between eslint and prettier configuration

## Motivation and Context
Pull request for issue #130 

## How Has This Been Tested?
Simply run the following command:

```
npm install
npm run lint
```

You should have currently one warning because one variable is not used

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -175,6 +175,8 @@
     }
   },
   "scripts": {
+    "lint": "eslint ./extension.js",
+    "lint:fix": "npm run lin -- --fix",
```
Comment (nhoizey) [inline package.json:179]: @rochejul should this be `npm run lint -- --fix`, with `lint` instead of `lin`?

**Comment by nhoizey** [comment]
@rochejul appart from the comment I made, this looks great, thanks!

**Comment by TheSench** [comment]
I messed up the merge (pasted `package.lock` over top of `package.json`) and am having issues editing this branch outside of GitHub (everything I do locally ends up creating separate branches).  Going to clean it up locally later this week.

**Comment by TheSench** [comment]
I have recreated this PR under #217 (still attributing rochejul for the changes, but addressing merge conflicts in package.json via rebasing).  Additionally, it's now updated to use ESLint 8 instead of 6.

**Pull request closed without merging**